javaCreateAccountCore/src/main/java/com/hamdan/forzenbook/java/createaccount/core/data/repository/InputException.java

```diff
@@ -0,0 +1,17 @@
+package com.hamdan.forzenbook.java.createaccount.core.data.repository;
+
+public class InputException extends Exception {
+    private final String message;
```

Comment:
Just make this not `final` then remove your second constructor. Not everything has to be final-- without a setter there's no way for a user to modify it